capstone/backend/src/main/java/com/google/spanner/LibraryFunctions.java

```diff
@@ -123,4 +135,61 @@ public static void createUser(long userId, String userName, String email)
     }
     return itemTypes;
   }
+
+  /*
+   * Provided a list of ItemTypes, this method will return a list of Stores
+   * that contain one or more of the Item Types provided.
+   *
+   * @param itemTypes list of Strings that describes the desired Item Types
+   * @return a list of all stores that have one or more Item Type
+   *
+   */
+  public static List<Store> getStoresWithItems(List<String> itemTypes) {
+    Map<Long, Store> stores = new HashMap<Long, Store>();
+    DatabaseClient dbClient = initClient();
+    String itemTypesString = getSQLList(itemTypes);
+    String query = "SELECT a.ItemId, a.ItemNameAndBrand, a.ItemType, b.StoreId, b.Price, c.Address, c.StoreName " +
```

Comment:
do you wanna try if this works:
```
Value itemListArray = Value.stringArray(itemTypes);
Statement statement =
        Statement.newBuilder(
                "SELECT a.ItemId......"
                    + "WHERE a.ItemType in UNNEST(@itemTypes)")
            .bind("itemTypes")
            .to(itemListArray)
            .build();
```

Then you won't need the `getSQLList` function